package.json

```diff
@@ -86,6 +86,7 @@
   "homepage": "https://github.com/SoftwareBrothers/adminjs#readme",
   "dependencies": {
     "@adminjs/design-system": "^4.1.0",
+    "@adminjs/themes": "^1.0.1",
```

Comment:
added as when using example repo https://github.com/SoftwareBrothers/adminjs-example-app fails when this package is linked and this package failed because this dependency does not exist

Reply:
This will result in a circular dependency issue because `adminjs` is a peer dependency of `@adminjs/themes`. `adminjs-example-app` is also an outdated repository.

Local linking issue can be solved by linking `@adminjs/themes` in your local app and in local `adminjs`

Reply:
good to know, I'll remove this package though